mapbox_tilesets/scripts/cli.py

```diff
@@ -529,6 +530,16 @@ def _upload_source(
     if replace:
         method = "put"
 
+    token_parts = mapbox_token.split(".")
+    if len(token_parts) > 1:
```

Comment:
should we raise an error here if len(token_parts) <= 1 ?

Reply:
Good point, since it can't be a legitimate token if it doesn't have three parts.